wordpress-mobile/WordPress-Android
# Issue/6511 google login error

#### Note
This should be merged ***after*** https://github.com/wordpress-mobile/WordPress-Android/pull/6671 and https://github.com/wordpress-mobile/WordPress-Android/pull/6672.

### Fix
Add error screen as described in https://github.com/wordpress-mobile/WordPress-Android/issues/6511.

### Test
0. Clear app data.
1. Tap ***Log In*** button.
2. Tap Google login view.
3. Turn off wifi and data.
4. Choose Google account.
5. Notice interface shown below.
![auth_i5_07_error](https://user-images.githubusercontent.com/3827611/29892202-f7bad0b2-8d8a-11e7-8bef-15c1d67f1f00.png)
